.pipelines/singletenancy/cilium-nodesubnet/cilium-nodesubnet-e2e-step-template.yaml

```diff
@@ -0,0 +1,196 @@
+parameters:
+  name: ""
+  clusterName: ""
+  scaleup: ""
+
+steps:
+  - task: AzureCLI@2
+    displayName: 'Update IP configs'
+    inputs:
+      azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+          set -e
+          clusterName=${{ parameters.clusterName }}
+          secondary_config_count=64
```

Comment:
Perhaps we can pair the `parameters.scaleup` to the amount secondary ipconfigs in a later PR?

Reply:
I wanted to do it in this PR itself, but I wasn't sure how the scaleup parameter was used. Does it indicate how many IPs we need (so secondary_config_count = scaleup), or is it some factor that defines the final IP count? If you could let me know I can make the change.

Reply:
`scaleup` is being used to bring up X number of pods in the cluster after node restart. Intent is to ensure E2E is more resilient to future changes. `scaleup` is being used to bring up X number of pods in the cluster after node restart. Intent is to ensure E2E is more resilient to future changes.